### PR TITLE
Polyfill for : HTMLSelectElement.prototype.selectedOptions

### PR DESCRIPTION
**Pull request by romainmenke**
fixes : #257

-----

_CI all green on my end_

### CODE REVIEW

File: polyfills/HTMLSelectElement/prototype/selectedOptions/config.toml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#### Aliases
+aliases = []
+
+#### Dependencies
+dependencies = [
+  "Object.defineProperty",
+  "document.querySelector"
+]
+
+notes = [ "Polyfill requires Object getters so fails in IE < 8" ]
```
Comment (romainmenke) [inline polyfills/HTMLSelectElement/prototype/selectedOptions/config.toml:10]: Will also add another node to clarify that `selectedOptions` does not return a live `HTMLCollection`

File: polyfills/URL/prototype/toJSON/tests.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* eslint-env mocha, browser */
```
Comment (JakeChampion) [inline polyfills/URL/prototype/toJSON/tests.js:1]: @romainmenke this has somehow also got the URL.prototype.toJSON changes in it, could you remove those as we already have a pull-request for that

File: polyfills/HTMLSelectElement/prototype/selectedOptions/config.toml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+#### Dependencies
+dependencies = [
+  "Object.defineProperty",
+  "document.querySelector"
+]
+
+notes = [
+  "Polyfill requires Object getters so fails in IE < 8",
+  "Polyfill does not return a live HTMLCollection, only a static NodeList that has the same methods as an HTMLCollection",
```
Comment (JakeChampion) [inline polyfills/HTMLSelectElement/prototype/selectedOptions/config.toml:12]: 👍

File: polyfills/URL/prototype/toJSON/tests.js (hunk 2 above)
Comment (romainmenke) [reply polyfills/URL/prototype/toJSON/tests.js]: @JakeChampion I merged the wrong branch, good catch!

**Review by JakeChampion — APPROVED** [review]
nice!

**Pull request merged**